cmd/manager/main.go

```diff
@@ -139,7 +144,29 @@ func (writer glogWriter) Write(data []byte) (n int, err error) {
 func main() {
 	defer glog.Flush()
 	cmd := newRootCommand()
-	err := cmd.Execute()
+
+	log.Debug("starting /metrics http handler")
+	http.Handle("/metrics", promhttp.Handler())
+	go http.ListenAndServe(":2112", nil)
+	log.Info("started /metrics http handler on port 2112")
+
+	metricsScheme := scheme.Scheme
```

Comment:
From the variable, you get the impression that you're using a scheme just for metrics, but it's really an alias for the global scheme.Scheme. It may be better just to use that without the alias so it's understood that you're changing the global one.

Reply:
Moving to the manager start actually let me use manager client and scheme so that cleans this up even more.